Use 2nd person in godot_notifications.rst (#11937)
godotengine/godot-docs: tutorials/best_practices/godot_notifications.rst

methods, but are still quite useful.

- :ref:`Node::NOTIFICATION_PARENTED <class_Node_constant_NOTIFICATION_PARENTED>`:

>   a callback that triggers anytime you add a child node to another node.

Reviewer: While the deed is done now and it's a good improvement, in these few cases it *may* have been genuinely beneficial to use passive verbs, instead, because these notifications can also be sent by the engine "doing its thing", not just as a result of user code.
```suggestion
  a callback that triggers anytime a child node is added to another node.
```
(also a "callback" that "triggers" are both *technically* the wrong terms for this but the rest of the tutorial sticks to it so, whatever)
